nemo/collections/nlp/data/language_modeling/megatron/bert_dataset.py

```diff
@@ -29,17 +29,17 @@
 class BertDataset(torch.utils.data.Dataset):
     def __init__(
         self,
-        name,
-        indexed_dataset,
-        data_prefix,
-        num_epochs,
-        max_num_samples,
-        masked_lm_prob,
-        max_seq_length,
-        short_seq_prob,
-        seed,
-        binary_head,
-        tokenizer,
+        name=None,
+        indexed_dataset=None,
+        data_prefix=None,
+        num_epochs=None,
+        max_num_samples=None,
+        masked_lm_prob=None,
+        max_seq_length=None,
+        short_seq_prob=None,
+        seed=None,
+        binary_head=None,
+        tokenizer=None,
```

Comment:
Why set these to None? Are they all optional? Could you also add type hints?

Reply:
I added the type hints. I added "=None" to convert them from positional arguments to keyword arguments.